ptxboa_streamlit.py

```diff
@@ -174,6 +174,7 @@
     "Market scanning",
     "Input data",
     "Deep-dive countries",
+    "Optimization",
```

Comment:
Turns out that this is culprit. If this line is commented out, the crash can be triggered by quickly switching between costs and optimization tab. If it is not commented out, the crash does not happen. 

I have no idea why, though.